docs/about/index.rst

```diff
@@ -64,8 +61,8 @@ Standard, see the :doc:`Governance <governance>` page.
 
 The initial development of the Beneficial Ownership Data Standard was
 funded by OpenOwnership as part of a UK Department for International
```

Comment:
Another 'OpenOwnership' to update. (Or maybe you left it as, historically, that was the org's name at the time?)

Reply:
Good catch. This was just an OpenOwnership that I missed